.github/workflows/ci.yml

```diff
@@ -10,7 +10,7 @@ on:
 
 env:
   CACHE_VERSION: 1
-  DEFAULT_PYTHON: 3.7
+  DEFAULT_PYTHON: 3.9
```

Comment:
Should we target 3.10 now?
```suggestion
  DEFAULT_PYTHON: "3.10"
```

Reply:
Note: zigpy is going to drop 3.7 too, but still target 3.8.

https://github.com/zigpy/zigpy-zigate/pull/131/commits/5f1fdc1aa8527fe9bb43b4210912125b78695e93

Reply:
Should we pin versions exactly like zigpy?
```suggestion
  DEFAULT_PYTHON: 3.9.14
```
https://github.com/zigpy/zigpy/pull/1050/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR10

Reply:
Yes

Reply:
We should match Zigpy’s targets

Reply:
Done.